packages/plugin-core/src/_extension.ts

```diff
@@ -496,6 +496,8 @@ export async function _activate(
       }
 
       ws.workspaceService = wsService;
+      // Start early to do this in background while Dendron initializes
+      const numContributors = wsService.getAllReposNumContributors();
```

Comment:
would be good to profile this command and report this to figure out if we need to cache these results or run into pathalogical cases for big repos. i would have the method return number of `ms` it executed for and then report it here

Reply:
I think we should also not `await` the `wsService.getAllReposNumContributors()` - let's put it in the `Promise.then(() => AnalyticsUtils.track())`.

Reply:
Thanks for the feedback, I:
- added a new event to track the contributor counts
- refactored tracking so initialization doesn't await it
- started tracking the duration of how long it took to compute

The actual computation is done by git, so even for large repos it shouldn't take too long (main Dendron repo takes about 42ms for me), but it's good to not have this block initialization anyway.